website/src/pages/auth/signin.tsx

```diff
@@ -176,32 +189,40 @@ const EmailSignInForm = ({
   };
   return (
     <form onSubmit={handleSubmit(signinWithEmail)}>
-      <Stack>
-        <Input
-          type="email"
-          data-cy="email-address"
-          variant="outline"
-          size="lg"
-          placeholder="Email Address"
-          {...register("email")}
-        />
-        {enableEmailSigninCaptcha && (
-          <CloudFlareCaptcha
-            siteKey={cloudflareCaptchaSiteKey}
-            options={{ size: "invisible" }}
-            ref={captcha}
-            onSuccess={() => setCaptchaSuccess(true)}
-          ></CloudFlareCaptcha>
-        )}
-        <SigninButton
-          data-cy="signin-email-button"
-          leftIcon={<Mail />}
-          mt="4"
-          disabled={!captchaSuccess && enableEmailSigninCaptcha}
-        >
-          Continue with Email
-        </SigninButton>
-      </Stack>
+      <FormControl isInvalid={errors.email ? true : false}>
+        <Stack>
+          <Input
+            type="email"
+            data-cy="email-address"
+            variant="outline"
+            size="lg"
+            placeholder="Email Address"
+            {...register("email", { required: true, pattern: /[^\s@]+@[^\s@]+\.[^\s@]+/g })}
+            isInvalid={errors.email ? true : false}
+            errorBorderColor="orange.600"
+          />
+          <FormErrorMessage>
+            {errors.email?.type === "required" && "Email is required"}
+            {errors.email?.type === "pattern" && "Email is invalid"}
```

Comment:
Can you localize this? eg create signin.json file

Reply:
Sorry I didn't understand this. What should this json file have, and where should I create it?

Reply:
I mean create file `/website/public/locales/en/signin.json` with simirlar following content or something:
```
{
  "email_error_required": "Email is required",
  "email_error_pattern": "Email is invalid"
}
```

If you still not sure how to do that, just ignore it for now.

Reply:
I can create these files but I'm still not sure how localization really works. Plus, If I'm not wrong the signin and edit page need translations all around, right? Maybe I can work on it later, but I think I will ignore it for now.